apps/web/src/pages/templates/hooks/useNavigateToStepEditor.ts

```diff
@@ -0,0 +1,28 @@
+import { useNavigate, useParams } from 'react-router-dom';
+import { StepTypeEnum } from '@novu/shared';
+
+import { useBasePath } from './useBasePath';
+
+export const useNavigateToStepEditor = () => {
+  const basePath = useBasePath();
+  const navigate = useNavigate();
+  const { channel, stepUuid, variantUuid } = useParams<{
+    channel: StepTypeEnum;
+    stepUuid: string;
+    variantUuid: string;
+  }>();
+
+  const navigateToStepEditor = () => {
+    const isVariant = !!variantUuid;
+    let navigatePath = `${basePath}/${channel}/${stepUuid}`;
+    if (isVariant) {
+      navigatePath += `/variants/${variantUuid}`;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** Is there a constant we can derive this from?‏

Reply:
Sorry, by that I meant something more like `/${VARIANTS_PATH}/${variantUuid}`